core/templates/dev/head/pages/exploration_editor/exploration_objective_editor_directive.html

```diff
@@ -0,0 +1,22 @@
+<p ng-class="{'has-error': !explorationObjectiveService.displayed || ((explorationObjectiveService.displayed.length < 15) || (explorationObjectiveService.displayed.length > 100))}">
```

Comment:
I don't think you need the double parens around the latter two conditions, right? You can just write A || B || C (without parens).